devtools/create_project/xcode.cpp

```diff
@@ -69,6 +69,31 @@ namespace CreateProjectTool {
 	_buildFile._flags = kSettingsSingleItem; \
 }
 
+/**
+ * Checks whether a file or directory exists.
+ */
+bool exists(const char * const fileName) {
+	struct stat unused;
+	return (stat(fileName, &unused) == 0);
+}
+
+const char * const DIRECTORIES[] = {
+	/* Homebrew */ "/usr/local/",
+	/* MacPorts */ "/opt/local/",
+	/* Fink     */ "/sw/"
+};
+
+std::string findPath(const char * const lib) {
+	for (int i = 0; i < 3; ++i) {
```

Comment:
Initially I wanted to disadvise usage of code from common etc. in tools, but it seems other tools do that too (*sigh*). You can either do `#include common/util.h` like other devtools. But it's a bit tricky because we don't actually link against any functionality from ScummVM itself. Thus, this works just because there's no dependencies. Or you just `#define` your own ARRAYSIZE like some other devtools like to do.